lib/publiccloud/ssh_interactive.pm

```diff
@@ -67,4 +67,12 @@ sub ssh_interactive_leave {
     $testapi::distri->set_standard_prompt('root');
 }
 
+sub ssh_interactive_post_fail {
+    select_console 'tunnel-console', await_console => 0;
+    send_key "ctrl-c";
+    send_key "ret";
+    assert_script_run('cd /root/terraform');
+    script_run('terraform destroy -no-color -auto-approve', 240);
+}
```

Comment:
I think we should return to the previous console. Otherwise the `post_fail_hook()` of "consoletest" might fail.

Reply:
I was also thinking about this but on this pase we run everything in `tunnel-console` which is the console intended for such 'preparations' and all the other consoles such as `root-console`, `user-console` or `log-console` are expecting us to have the SSH serial console activated. See. line 830 of `/var/lib/openqa/share/tests/sle/lib/susedistribution.pm` file.